libsolidity/interface/CompilerStack.cpp

```diff
@@ -1244,15 +1245,16 @@ void CompilerStack::assemble(
 	if (
 		m_evmVersion >= langutil::EVMVersion::spuriousDragon() &&
 		compiledContract.runtimeObject.bytecode.size() > 0x6000
-	)
+	   )	
 		m_errorReporter.warning(
 			5574_error,
 			_contract.location(),
-			"Contract code size exceeds 24576 bytes (a limit introduced in Spurious Dragon). "
+			string("Contract code size is ") +
```

Comment:
This is a more compact syntax for creating a string from a literal:
```suggestion
			"Contract code size is "s +
```